doc/integrator/database.rst

```diff
@@ -26,10 +26,13 @@ integration and production => ``main_2018``
 
 The schema ``main_2017`` still exists to be able to rollback the production.
 
-To do that we should manage the ``PGSCHEMA`` and the ``PGSCHEMA_STATIC`` variable in your makefiles.
+To do that we should manage the ``PGSCHEMA`` and the ``DOCKER_PGSCHEMA_STATIC`` variable in your
+makefiles.
 
-The ``PGSCHEMA`` should be set in the ``Makefile`` and the ``PGSCHEMA_STATIC`` for the production should be
-set in a specific makefile for the production e.-g. ``production.mk``.
+The ``PGSCHEMA`` should be set in the ``Makefile`` and the ``DOCKER_PGSCHEMA_STATIC`` for the
+production should be set in a specific makefile for the production e.-g. ``production.mk``.
+
+And the line `PGSCHEMA=${docker_schema}` should be removed from your `.env.mako` file.
 
 To to the schema we provide a Postgres function, to create it:
```

Comment:
Something is wrong with this sentence. Suggestion "To create the schema, we provide a Postgres function:"

Reply:
Updated :-)